src/main/java/net/minecraftforge/client/gui/ModListScreen.java

```diff
@@ -451,7 +453,7 @@ public void upload() {
 
         if (vercheck.status() == VersionChecker.Status.OUTDATED || vercheck.status() == VersionChecker.Status.BETA_OUTDATED)
             lines.add(ForgeI18n.parseMessage("fml.menu.mods.info.updateavailable", vercheck.url() == null ? "" : vercheck.url()));
-        lines.add(ForgeI18n.parseMessage("fml.menu.mods.info.license", selectedMod.getOwningFile().getLicense()));
```

Comment:
Why is this cast necessary? It would imply that there is a breaking change here requiring a cast.